keyboards/ano/info.json

```
@@ -0,0 +1,10 @@
+{
+    "keyboard_name": "Ano", 
+    "url": "https://github.com/sauvehoo", 
+    "maintainer": "sauvehoo", 
+    "layouts": {
+        "LAYOUT_ano_6x18": {
```

This must be a case-sensitive match for the name used in `ano.h`.

```suggestion
        "LAYOUT_all": {
```

It's also missing two key objects, but let's deal with that later.